app/views/users/index.html.erb

```diff
@@ -1 +1,15 @@
-<h1>List of users</h1>
\ No newline at end of file
+<section>
+  <% @users.each do |user| %>
+    <div class="d-flex justify-content-between">
+      <div>
+        <img src="<%= user.photo %>" alt='User Avatar' style="width:20%" />
+      </div>
+      <div class='flex-grow-1 d-flex justify-content-between align-items-center'>
+        <a href="/users/<%= user.id %>">
+          <h1><%= user.name %></h1>
+        </a>
+        <p>Number of posts: <%= user.posts_counter || 0 %></p>
+      </div>
+    </div>
+  <%end%>
```

Comment:
## [OPTIONAL]

- [ ] Kindly I would suggest you a better implementation that will make your code simple and easier to read, it's a better practice to use partial to render user. Example :point_down: 

```ruby
    # app/views/users/_user.html.erb

     #   HTML Code Here displaying
     #   user.photo, user.name and user.posts_counter

    # app/views/users/index.html.erb

    <%= render @users %>
```

Reply:
Thanks @andersonlebon 
I will do it